.github/workflows/quality-assurance.yml

```diff
@@ -85,22 +86,22 @@ jobs:
         run: ATRIUM_ANDROID_JAR="$PWD/android-jar-cache/android.jar" ./gradlew checkDexer
 
   forwardCompatibility:
-    needs: dexer
+    needs: []  #Not sure if it should be left empty or if it should contain generate-matrix
```

Comment:
if you would want to use different OS etc. then you would need it. IMO good enough if we use jdk17 and adopt only for now